src/features/blocks/BlocksToolbox.tsx

```diff
@@ -0,0 +1,16 @@
+import './BlocksToolbox.css'
+import {ToolboxConfiguration} from "../../toolboxconfig"
+import BlocksCategory from './BlocksCategory'
+
+function BlocksToolbox({categories}: ToolboxConfiguration) {
+  return (
+    <div className={'toolbox'}>
+      {categories.map(category => {
+        return <BlocksCategory key={category.id} name={category.name} blocks={category.blocks}
+                               highlight={category.highlight} id={category.id}/>
```

Comment:
It would be better for code readibility to be like this:

    <BlocksCategory
        prop1={value1}
        prop2={value2}
        prop3={value3}
    />

I suggest you to use this syntax as soon as you have 3 or more props, it's also quicker to edit and remove properties with WebStorm this way.

Reply:
Understood & changed, thanks!